icons/book-headphones.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
Audio books are great usecase! Seeing this icon I was not directly thinking of audio books, so nice one!